syncstorage-spanner/src/models.rs

```diff
@@ -1842,10 +1842,7 @@ impl SpannerDb {
             sqlparam_types.insert("payload".to_owned(), payload.spanner_type());
             sqlparams.insert("payload".to_string(), payload.into_spanner_value());
             let now_millis = timestamp.as_i64();
-            let ttl = bso.ttl.map_or(i64::from(DEFAULT_BSO_TTL), |ttl| {
-                ttl.try_into()
-                    .expect("Could not get ttl in put_bso_async_test")
-            }) * 1000;
+            let ttl = bso.ttl.map_or(i64::from(DEFAULT_BSO_TTL), |ttl| ttl.into()) * 1000;
```

Comment:
Why are we dropping the `expect`?

Reply:
Because `try_into` -> `into`, I guess the `try_into` wasn't necessary (maybe the new clippy caught that?)

Reply:
Yup! Looks like there is a [new clippy lint](https://rust-lang.github.io/rust-clippy/master/index.html#/unnecessary_fallible_conversions) that catches uses of `try_into` and `try_from` when `From<T>` and `Into<T>` are implemented